rebase.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+git checkout --theirs . & git add . & git rebase --continue
```

Comment:
seems like this should be a hack script.  Don't think it should live top-level.

Reply:
This was accidentally committed. Will remove